packages/terra-graphs/docs/controls/Line.md

```diff
@@ -115,15 +115,21 @@ lineDefault.loadContent(Carbon.api.line(data));
 ```
 
 #### Multi Line
-
-For loading multiple data-sets, you can load as additional content:
-
+Adding a single Data set can be done by creating a line with a [Data](#Data) set.
 ```javascript
 var lineDefault = Carbon.api.graph(/* Input JSON */);
 lineDefault.loadContent(Carbon.api.line(/* Data array A */));
-lineDefault.loadContent(Carbon.api.line(/* Data array B */));
-lineDefault.loadContent(Carbon.api.line(/* Data array C */));
-lineDefault.loadContent(Carbon.api.line(/* Data array D */));
+```
+
+For loading multiple data-sets, you can load as additional content as an array:
```

Comment:
nit: you can load ~~as~~ additional content as an array:

Reply:
good eye, fixed [here](https://github.com/cerner/terra-graphs/pull/29/commits/eb4ca954c9370b815d6ea163b5d977d7754a6feb)